enhancements/sig-policy/78-policy-collection/README.md

```diff
@@ -0,0 +1,77 @@
+# Policy Collection
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The goal of this feature is to automate and simplify deployment of policies, policy sets and
+policy generator projects by creating (transferring) a community that provides best practices, samples, blogs
+and deployment resources.  The repository for this community is
+[policy-collection](https://github.com/open-cluster-management-io/policy-collection/).
+
+## Motivation
+
+The governance policy framework is currently provided without any samples or guidance on
+how to create policies other than by detailing the architecture of the 
+[policy framework](https://open-cluster-management.io/getting-started/integration/policy-framework/).
+While this is functional, it does not lend itself to easy adoption by new community users.
+
+### Goals
+
+- Transfer the policy collection community that provides samples, best practices and encourages quicker
+  adoption of the policy framework.
+- Make sure the transferred community is easy to consume by first time users of ocm policy.
+
+### Non-Goals
+
+- Beecause policies can be intended to work in many different environments with prerequisites that
+  may not be available, validation of policies is focused on syntax initially and additional validation
+  can be considered in the future.
+- At this time a procedure to provide support for donated policies by moving them to a stable
```

Comment:
not quite clear about what is the criteria to address this.

Reply:
I am removing that item since I agree that it is confusing to provide as a non goal.